src/frontend/jsonapi/basics.md

```diff
@@ -383,13 +409,13 @@ By default, only the first 25 items are returned if nothing else is specified. T
     });
     ```
 
-Thus, you will get the first 2 items instead. If more items are available, the first response automatically contains links for navigating through the results:
+This will get only the first 2 items. If more items are available, the first response automatically contains links for navigating through the results:
 
 ```json
 "links": {
     "first": "http://localhost:8000/jsonapi/product?page%5Boffset%5D=0&page%5Blimit%5D=2",
     "prev": "http://localhost:8000/jsonapi/product?page%5Boffset%5D=0&page%5Blimit%5D=2",
-    "next": "http://localhost:8000/jsonapi/product?page%5Boffset%5D=4&page%5Blimit%5D=2",
+    "next": "http://localhost:8000/jsonapi/product?page%5Boffset%5D=2&page%5Blimit%5D=2",
```

Comment:
If the "self" link contains offset=2, then the "next" link must contain offset=4